build/dsc.rake

```diff
@@ -80,7 +94,9 @@ eod
     task :clean, [:dsc_resources_path] do |t, args|
       dsc_resources_path = args[:dsc_resources_path] || default_dsc_resources_path
       puts "Cleaning #{item_name}"
-      FileUtils.rm_rf "#{default_dsc_resources_path}"
+      FileUtils.rm_rf "#{dsc_resources_path}"
+      FileUtils.rm_rf "#{vendor_dsc_resources_path}"
```

Comment:
@jpogran this line in particular was necessary to properly remove vendored resources prior to revendoring